src/formulas/compiler.ts

```diff
@@ -22,7 +22,7 @@ const OPERATOR_MAP = {
   "<=": "LTE",
   "<": "LT",
   "^": "POWER",
-  "&": "CONCATENATE",
+  "&": "CONCAT",
```

Comment:
What is the concrete difference of behaviour of the 2 functions? (concat vs concatenate)

Reply:
`CONCATENATE` takes ranges and not single arguments as input. Vectorization is only possible with simple arguments.

Besides, that's probably why both exist, they both do the same thing, but one will work as an aggregator and the other will vectorize